scripts/sync-ros2-gbp-devel-branch.py

```diff
@@ -0,0 +1,219 @@
+# This is a program to sync rosdistro source entries with the release repository devel_branch.
```

Comment:
Based on the licensing discussion in #26055 this should probably get a copyright header.

Reply:
Good call, will add.